core/src/main/scala/chisel3/internal/MonoConnect.scala

```diff
@@ -190,6 +206,138 @@ private[chisel3] object MonoConnect {
       case (sink, source) => throw MismatchedException(sink, source)
     }
 
+  /** Check [[Aggregate]] visibility. */
+  private[chisel3] def aggregateConnectContextCheck(
```

Comment:
What does `true` mean and what does `false` mean? It would be helpful if you could expand this comment to say that it checks contexts (and throws exceptions) as well as with the information about what the return value means.